client/app/components/dynamic-table/index.js

```diff
@@ -12,8 +12,8 @@ function DynamicTable($sanitize) {
   this.orderByReverse = false;
 
   this.pageChanged = () => {
-    const first = this.count * (this.page - 1);
-    const last = this.count * (this.page);
+    const first = this.itemsPerPage * (this.page - 1);
+    const last = this.itemsPerPage * (this.page);
```

Comment:
I'm not sure why we have both, but we [set them both to the same value](https://github.com/rockwotj/redash/blob/bc3691b5aedf5ec069690c27274953e01b083ee7/client/app/components/dynamic-table/index.js#L8). Why would this make a difference?